src/features/bumpkins/components/Trade.tsx

```diff
@@ -100,7 +101,18 @@ const ListTrade: React.FC<{
   if (!selected) {
     return (
       <div className="space-y-2">
-        <div className="pl-2 py-2">
+        <div className="pl-1 py-2 flex items-center">
+          <img
+            src={SUNNYSIDE.icons.arrow_left}
+            className="self-start cursor-pointer mr-3"
+            style={{
+              top: `${PIXEL_SCALE * 2}px`,
+              left: `${PIXEL_SCALE * 2}px`,
+              width: `${PIXEL_SCALE * 11}px`,
+            }}
+            alt="back"
+            onClick={onBack}
```

Comment:
You can use `onCancel` here instead of creating onBack since both of them are directing to the same function